app/assets/javascripts/hidden_values.js

```diff
@@ -3,14 +3,15 @@ function turn_textarea_switch(checkbox) {
   var id = checkbox.id.replace(/hidden_value$/, "value");
   var source = document.getElementById(id);
   var $editorContainer = $('.editor-container');
+  var value = secureFilters.html(source.value);
 
   if (checkbox.checked) {
-    target = '<input class="form-control" type="password" id="' + id + '" name="' + source.name + '" value ="' + source.value + '"></input>'
+    target = '<input class="form-control" type="password" id="' + id + '" name="' + source.name + '" value ="' + value + '"></input>'
```

Comment:
Normally we just call `.val()` to set the value, rather than HTML escaping it.

(Incidentally, I think underscore.js provides an escape function, we added it recently for that purpose.)